openedx/core/djangoapps/schedules/management/commands/tests/test_send_course_update.py

```diff
@@ -35,3 +42,13 @@ def setUp(self):
         mock_highlights = patcher.start()
         mock_highlights.return_value = ['Highlight {}'.format(num + 1) for num in range(3)]
         self.addCleanup(patcher.stop)
+
+    @ddt.data(
+        ExperienceTest(experience=ScheduleExperience.EXPERIENCES.default, offset=expected_offsets[0], email_sent=False),
+        ExperienceTest(experience=ScheduleExperience.EXPERIENCES.course_updates, offset=expected_offsets[0], email_sent=True),
+    )
+    def test_schedule_in_different_experience(self, test_config):
+        self._check_if_email_sent_for_experience(test_config)
+
+    def test_no_experience_specified(self):
```

Comment:
Nit: can each of these `test_no_experience_specified` tests just be implemented as an `ExperienceTest` `ddt` case, with `experience=None`?

Reply:
yup, done!